src/components/templates/FireBarrel/FireSeat.tsx

```diff
@@ -122,8 +43,10 @@ const FireSeat: React.FC<PROPS> = ({ person, chairNumber, roomName }) => {
         show={!!videoError}
         onHide={() => setVideoError("")}
         errorMessage={videoError}
-        onRetry={connectToVideoRoom}
+        // onRetry={connectToVideoRoom}
+        onRetry={() => {}}
         onBack={() => {}}
+        // onBack={() => (setSeatedAtTable ? leaveSeat() : setVideoError(""))}
```

Comment:
curious what the eventual plan is for these?

Reply:
Unsure, I just didn't want to wholesale delete code. Just needed it out of the way to avoid errors.